ca/certificate-authority.go

```diff
@@ -64,6 +67,22 @@ type PKCS11Config struct {
 	Label  string
 }
 
+// The maximum number of requests to allow in the signing queue
+// before we start returning Service Unavailable errors in response
+// to new-certificate requests.
+const signingQueueSize = 100
```

Comment:
Is this the right value?  Who knows?  Probably could be an order of magnitude or so smaller.